static/js/publisher/tour/tourOverlay.js

```diff
@@ -71,13 +94,14 @@ export default function TourOverlay({ steps, onHideClick }) {
   // when current step changes, scroll step element into view
   useEffect(
     () => {
-      step.el.scrollIntoView({ block: "center", behavior: "smooth" });
+      // TODO: consider scrolling based on mask position instead of element
+      step.elements[0].scrollIntoView({ block: "center", behavior: "smooth" });
```

Comment:
This causes highlighted content to appear below the viewport if it's taller than half the viewport height.